README.md

```diff
@@ -29,7 +29,7 @@ If you are trying to access grpc-go from within China, please see the
 
 Prerequisites
 -------------
-gRPC-Go requires Go 1.9 or later.
+gRPC-Go supports the three latest releases of Go.
```

Comment:
It might be a good idea to link "latest releases of Go" to https://golang.org/doc/devel/release.html. Also might you want to qualify the statement with as "officially supports"?

Reply:
See #3526